modules/wfa-qt/Kconfig

```diff
@@ -32,3 +32,9 @@ config WPAS_READY_TIMEOUT_MS
 	default 10000
 	help
 	  Set the timeout for WPA supplicant to be ready.
+
+config WFA_QT_REBOOT_TIMEOUT
```

Comment:
if you add `_MS` then it makes it obvious that it's milliseconds, without need to read the help.